src/app/projects/projects.jsx

```diff
@@ -31,75 +36,11 @@ export default function Projects() {
 
   return (
     <>
-      <div className="faq-main">
+      <div className="projects-main">
         <ProjectsSectionStart
           sectionType={SectionType.light}
         ></ProjectsSectionStart>
       </div>
     </>
   );
 }
-
-/**
- * Fetches project data from github
- *
- * This will look through each repository for a meta.yml file
- * If the project has this file, it will gather all project information
- * needed to populate the ProjectCard component
- *
- * Note: Github has a rate limit (at time of writing) of 60 requests per hour per ip address
- * for unauthenticated requests.  This means there's a chance that this page will not load properly
- * If the user is refreshing frequently.
- *
- * It may be useful to cache this data and only re-fetch on longer intervals if this becomes a frequent issue.
- * See https://swr.vercel.app/docs/advanced/cache#localstorage-based-persistent-cache
- *
- * @param  fetch args
- * @returns array of data needed for project cards
- */
-const fetchGithubProjectData = async (...args) => {
```

Comment:
I moved these functions to github.js and made them simpler and easier to use and compose together.  This made it easier to re-use the fetch single project data in multiple places without duplicating logic.